### ISSUE TITLE
Clicking Links crashes Firefox

### ISSUE THREAD

**Issue by kstev99** [sneakypete81/updatescanner#328]
This problem just began this morning after a daily Firefox **NIGHTLY** (2019-02-15) update on Windows 10.  Notification of a changed website occurs.  I open the Sidebar and open the changed webpage.  

Clicking any link on that page or the Update Scanner header at the top of the page crashes the browser.
I was using the latest Beta version of Update Scanner (4.3.3), but I reverted back to 4.3.2 when this problem began, but it did not fix the problem.

**Comment by sneakypete81** [comment]
Thanks for your report. Does this happen for all websites? Does Firefox Beta work for you?

Please try with a new Firefox profile too.

**Comment by kstev99** [comment]
Thanks for the quick response. YES it happens for all websites in my sidebar.  I did however create a New Firefox Nightly profile and with Update Scanner as the ONLY add-on installed the problem does not happen.  Leads me to believe there is a conflict with another add-on.  I will begin enabling each one on the new profile and see if I can find it.

**Comment by kstev99** [comment]
I installed all of my add-ons in the new Firefox profile which had only Update Scanner installed.
I also disabled all existing add-ons in my working FF profile and began enabling them one at a time.

In both profiles, once I added the **Roboform** add-on it would cause Firefox to crash every time.
[https://addons.mozilla.org/en-US/firefox/addon/roboform/](url)

Note that clicking the website in the sidebar does not crash.  Only when I click the Update Scanner header of the checked page that opens, OR if I try clicking one of the highlighted yellow links that have changed
